frontend/tsconfig.json

```
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "noEmit": true,
+
+    "allowSyntheticDefaultImports": true,
+    "forceConsistentCasingInFileNames": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "strict": true,
```

Afaik [JSON does not allow trailing commas](https://stackoverflow.com/a/201856). There are a few places where these can be found:

```suggestion
    "strict": true
```

---

That's true, but the program parsing `tsconfig.json` certainly accepts this. There are many programs consuming JSON which allow trailing commas and comments, despite them not being in the JSON specs. I don't care, I can remove all trailing commas if you want, but I don't really see the benefit of doing so, as long as the config files can be read.

---

I tend to not rely on unspecified and undefined behavior of specific parsers. That's something which can easily break at any time. Plus, my editor complains…

---

Fair point. I removed the commas in JSON files.